src/bin/pg_autoctl/defaults.h

```diff
@@ -51,6 +51,10 @@
 #define POSTGRESQL_FAILS_TO_START_TIMEOUT 20
 #define POSTGRESQL_FAILS_TO_START_RETRIES 3
 
+#define FAILOVER_FORMATION_NUMBER_SYNC_STANBYS 1
+#define FAILOVER_NODE_CANDIDATE_PRIORITY 100
+#define FAILOVER_NODE_REPLICATION_QUORUM 1
```

Comment:
In the PostgresSetup structure the slot NodeReplicationSettings is a boolean. We use this default value for filling-in this slot, if I'm following the PR correctly. The default reads as an integer rather than a boolean, is it on purpose?

Reply:
it needs to be bool, fixed